index.js

```diff
@@ -231,16 +239,25 @@ const init = async () => {
     });
   });
 
-  // Then we load events, which will include our message and ready event.
-  const evtFiles = await readdir("./events/");
-  evtFiles.forEach(file => {
-    const eventName = file.split(".")[0];
-    client.logger.log(`Loading Event: ${eventName}. 👌`, "log");
-    const event = new (require(`./events/${file}`))(client);
-    // This line is awesome by the way. Just sayin'.
-    client.on(eventName, (...args) => event.run(...args));
-    delete require.cache[require.resolve(`./events/${file}`)];
-  });
+  const eventList = [];
```

Comment:
Not exactly sure why you switched to using klaw as events should not have sub-folders.

Reply:
Removed klaw in upcoming commit.